### DISCUSSION TITLE
Support for Custom Defined Nuget Sources

### DISCUSSION THREAD

**Discussion by chrisbewz** [tareqimbasher/NetPad#284] [Q&A] [answered]
Hello, i found recently NetPad while searching for alternatives to LinqPad and its being so much useful. 

But I cant figure out how can define a custom nuget source to use my private packages for some tests instead manually referencing my private project assemblies through script properties.

If its isnt supported for now, are there any plans to make it available?

Thanks

**Comment by tareqimbasher** [comment, marked as answer]
Sorry I must've missed this. You have to add custom nuget feeds manually by editing your `Nuget.config` file. You can [see this](https://github.com/tareqimbasher/NetPad/blob/main/CONTRIBUTING.md#web-browser) for where that file is on your system.

Once you've added your custom feed, NetPad will pick it up and you can start searching and installing packages from that custom feed.

I'm really glad you like NetPad, thank you for sharing.
